docs/docs/20-quickstart.md

```diff
@@ -508,52 +510,28 @@ the previous section.
     EOF
     ```
     
-  :::info
-  Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
-  :::
-
-1. Use the CLI to view our `Warehouse` resource:
-
-    ```shell
-    kargo get warehouses --project kargo-demo
-    ```
+      :::info
+      Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
+      :::
```

Comment:
Let's ditch this. I put this here, but it feels out of place now.

We can move this to be inline in the Go/CRD docs. (You can make a separate issue for that.)